source/command_bus.coffee

```diff
@@ -7,19 +7,22 @@ class Space.messaging.CommandBus extends Space.Object
   }
 
   _handlers: null
+  _onSendHooks: null
 
   constructor: ->
     super
     @_handlers = {}
+    @_onSendHooks = []
 
-  send: (command, callback) ->
+  send: (command, callback, options={}) ->
     if @meteor.isServer
       # ON THE SERVER
       handler = @_handlers[command.typeName()]
-      if not handler?
+      hook(command) for hook in @_onSendHooks when !options.ignoreHooks
```

Comment:
Yeah, the problem we have here is that this only needed in some very specific test cases + it needs to be done on a per-sending basis. it would not work if you "configure" that all messages should bypass hooks. I agree that this should work differently, as it is only testing related and should not be part of the public api.